### ISSUE TITLE
Why was the number "progress bar" removed for fread?

### ISSUE THREAD

**Issue by eantonya** [Rdatatable/data.table#3050]
Just curious why the number progress bar was removed and replaced with the "graphical" progress bar?

Seems to me that this representation shows less info + is fragile to hitting Enter.

**Comment by HughParsonage** [comment]
My fault: https://github.com/Rdatatable/data.table/issues/2481

https://github.com/Rdatatable/data.table/issues/2481#issuecomment-348690091

**Comment by eantonya** [comment]
@mattdowle I personally don't care about the visual representation, and if people think this is prettier - that's fine with me, but maybe we can have the number printed as well below it or next to it to have more informative output.

**Comment by mattdowle** [comment]
@eantonya whoa - where did I mention pretty?  I did the `=` to avoid the `\r`, that was all.  Nothing to do with pretty.  I started my reply above positively and explained the background to it too. That bug in fread was fraught and lasted weeks related to the multi-threading. It appeared to be Windows-only RStudio-only but turned out to be neither.  At the end of the day there is a thread printing that progress inside a parallel region, and calling the R API (printing) is not thread-safe.  I think it should be fine to try the `\r` again but at one point I thought it wouldn't be possible to even have a text progress at all inside the parallel region.

**Comment by eantonya** [comment]
@mattdowle haha, I didn't mean you personally find it pretty - I was simply attempting to cover the case of some people enjoying the current representation. But perhaps I've misjudged, and that group is of size 0.

**Comment by mattdowle** [comment]
yep I can't imagine anyone finds it useful.  No bounds are even printed so you don't know how many `=` are going to printed -- you just know it is progressing and that's all.  We can try the % again.

**Comment by MichaelChirico** [comment]
Not much interest in including the % it seems, 7 years later. Closing.
